Lib/test/test_descr.py

```diff
@@ -845,6 +847,14 @@ class Module(types.ModuleType, str):
             self.fail("inheriting from ModuleType and str at the same time "
                       "should fail")
 
+        # Issue 34805: Verify that definition order is retained
```

Comment:
Small nit (feel free to ignore): I suggest to use "bpo" instead of "issue" for disambiguating if we ever migrate to GitHub issues.


```suggestion
        # bpo-34805: Verify that definition order is retained
```